packages/utils/src/dom/html.ts

```diff
@@ -56,13 +56,14 @@ function simpleHtmlParse (html, documentContext) {
     // (possibly a text node) in front of the comment. So, KO does not attempt to workaround this IE issue automatically at present.
 
     // Trim whitespace, otherwise indexOf won't work as expected
-  var tags = stringTrim(html).toLowerCase(), div = documentContext.createElement('div'),
+  let div : any = documentContext.createElement('div')
```

Comment:
better as `const`

Reply:
See https://github.com/Auge19/tko/pull/6 for all the var-to-let-to-const auto-fixes